build/buildspec.yaml

```diff
@@ -13,9 +14,10 @@ phases:
       - pip install --upgrade pip
       - pip install --upgrade setuptools wheel
       - pip install twine
+      - echo $PYPI_PASSWORD | keyring set https://pypi.org/ $PYPI_USERNAME
```

Comment:
still doesn't seem secure to `echo` out the credentials here, we should be able to set the `parameter-store` secret to have it load as part of the pypi config natively...